apis/Google.Cloud.EntityFrameworkCore.Spanner/Google.Cloud.EntityFrameworkCore.Spanner.Snippets/coverage.xml

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You can delete this file - if you fetch from master again, there's a change which will prevent this from being generated. (With this in place, the tests will fail to run.)

Reply:
hmm its still being generated for me with a rebase from master.  I'm looking into it.

Reply:
25e0c80ac634172e6556b4d42286a6ec7cc6e9f2 suppresses generation of the csproj, but possibly not the coverage.xml.  I'll see about adding that.

Reply:
Gah - really sorry about that. I must have not rerun it after thinking I'd fixed it :(